runtimes/alibi-detect/mlserver_alibi_detect/runtime.py

```diff
@@ -136,6 +141,18 @@ def _detect(self, payload: InferenceRequest) -> InferenceResponse:
 
         return self._encode_response(self._postproc_pred(pred))
 
+    def _log_metrics(self, current_pred: dict) -> None:
+        if "data" not in current_pred:
+            return
+
+        data = current_pred["data"]
+        if "is_drift" in data:
```

Comment:
Question for @adriangonz and @joshsgoldstein. What is the motivation for this PR? 

Asking as wondering if we are definitely only concerned about `is_drift` here? Often `p_val` and `distance` are of equal (or greater) interest. As is `threshold`, and `distance_threshold` (dependent on detector).

Reply:
@ascillitoe we are enabling scv2 alerting in deploy